docs/using-getting-started.asciidoc

```diff
@@ -0,0 +1,304 @@
+
+| message
+| GET /content HTTP/1.1" 200 2571 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.106 Safari/537.36
+
+// ==============================================================
+
+| source.address
+| 10.42.42.42
+
+// ==============================================================
+
+| source.ip
+| 10.42.42.42
+
+// ==============================================================
+
+| url.original
+| `/content`
+
+// ==============================================================
+
+| user_agent.original
+| `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.106 Safari/537.36`
+
+|=====
+
+Beyond extracting values that are present in the original event, we also populate
+other fields to provide additional context about the event itself.
+
+* `ecs.version`: States which version of ECS the ingest pipeline was developed against.
+* `event.dataset` and `event.module`: Answers "where is this event from" and are expected to have a
+  hardcoded value per pipeline, per source.
+* `event.kind`, `event.category`, `event.type`, and `event.outcome`: The https://www.elastic.co/guide/en/ecs/current/ecs-category-field-values-reference.html[categorization fields]
+  should also be hardcoded using knowledge of each type of event the source emits. The contents of
+  these fields are limited to the specifically allowed values detailed in the ECS documentation.
+
+[options="header"]
+|=====
+| Field Name | Value
+
+// ===============================================================
+
+| ecs.version
+| 1.5.0
+
+// ==============================================================
+
+| event.module
+| apache
+
+// ===============================================================
+
+| event.dataset
+| apache.access
+
+// ==============================================================
+
+| event.kind
+| event
+
+// ===============================================================
+
+| event.category
+| [ "web", "access" ]
+
+// ==============================================================
+
+| event.outcome
+| success
+
+|=====
+
+Lastly, existing field values can be interpreted or enriched using a processor, with the results
+populating additional fields in the final event.
+
+* The `user_agent` processor extracts details from the original user agent string, `user_agent.original`.
+* IP fields like `source.ip` can provide enrichment using the `geopip` processor to add information about the
+  location and autonomous system number (ASN) associated with an IP address.
+* The `registered domain` processor reads a field containing a hostname and writes the registered domain to
+  another field
+* Event collectors, such as https://www.elastic.co/guide/en/beats/libbeat/current/beats-reference.html[Beats], can enrich
+  each event with metadata from the machine's hosting provider (cloud) and/or from the host machine (host).
+
```

Comment:
Perhaps preface this final table with something like

> Here are some examples of additional fields added by metadata or parser processors.

Also I think we could trim the list of fields we list in the table, to keep it short. A few of each is enough to illustrate, let's keep those that will speak to people most. Perhaps remove

* host.mac
* host.os.build
* either host.os.family or host.os.platform
* user_agent.device.name

Reply:
Good idea. I was worried the third table was maybe going a little _too_ deep for a getting started section, but I think scaling back is a good way to help address. 👍